### ISSUE TITLE
Can't create new Jobgroup

### ISSUE THREAD

**Issue by coderDem** [IteraSpeed/OpenSpeedMonitor#12]
Hello,

after the last update to the app version 3.1.3, I can't add new jobgroup.
When I try to add a new jobgroup I get this errormessage: 

```javascript
Error processing GroovyPageView Error processing GroovyPageView /jobGroup/create has problem at line 530
```

And here the Stacktrace:
```javascript
org.codehaus.groovy.grails.web.pages.exceptions.GroovyPagesException: Error processing GroovyPageView: Error executing tag <g:form>: Error executing tag <g:render>: Unknown name value for enum class de.iteratec.osm.measurement.environment.wptserverproxy.Protocol: 

	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)

	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)

	at java.lang.Thread.run(Thread.java:745)

Caused by: org.codehaus.groovy.grails.web.taglib.exceptions.GrailsTagException: Error executing tag <g:form>: Error executing tag <g:render>: Unknown name value for enum class de.iteratec.osm.measurement.environment.wptserverproxy.Protocol: 

	at jobGroup_create$_run_closure2.doCall(jobGroup_create:53)

	at jobGroup_create.run(jobGroup_create:56)

	... 3 more

Caused by: org.codehaus.groovy.grails.web.taglib.exceptions.GrailsTagException: Error executing tag <g:render>: Unknown name value for enum class de.iteratec.osm.measurement.environment.wptserverproxy.Protocol: 

	at jobGroup_create$_run_closure2_closure6.doCall(jobGroup_create:46)

	... 5 more

Caused by: java.lang.IllegalArgumentException: Unknown name value for enum class de.iteratec.osm.measurement.environment.wptserverproxy.Protocol: 

	at jobGroup__form_gsp.run(jobGroup__form_gsp:32)

	... 6 more

Caused by: java.lang.IllegalArgumentException: No enum constant de.iteratec.osm.measurement.environment.wptserverproxy.Protocol.

	at java.lang.Enum.valueOf(Enum.java:236)

	... 7 more
```

Maybee you now, what the problem is.

best regrads
coderDem

**Comment by coderDem** [comment]
Ok the problem come from this commit: https://github.com/IteraSpeed/OpenSpeedMonitor/commit/6244912b482a6c9a32276cfcd514a6f6caf07219

I reset my version to the commit 83c86d8.

**Comment by nilskuhn** [comment]
This is fixed with  version 3.2
